tests/std/tests/concepts_latest_matrix.lst

```diff
@@ -3,23 +3,36 @@
 
 RUNALL_INCLUDE .\prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/w14640 /Zc:threadSafeInit- /EHsc /std:c++latest"
+*	PM_CL="/w14640 /Zc:threadSafeInit- /EHsc /std:c++latest"
 RUNALL_CROSSLIST
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=0 /permissive- /Zc:noexceptTypes-"
+ASAN	PM_CL="/MD /permissive- /Zc:noexceptTypes- -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=1 /permissive-"
+ASAN	PM_CL="/MD /permissive- -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MD /D_ITERATOR_DEBUG_LEVEL=0 /permissive- /Zc:char8_t- /Zc:preprocessor"
+ASAN	PM_CL="/MD /permissive- /Zc:char8_t- /Zc:preprocessor -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=0 /permissive- /Zc:wchar_t-"
+ASAN	PM_CL="/MDd /permissive- /Zc:wchar_t- -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=1 /permissive-"
+ASAN	PM_CL="/MDd /permissive- -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MDd /D_ITERATOR_DEBUG_LEVEL=2 /permissive- /fp:except /Zc:preprocessor"
+ASAN	PM_CL="/MDd /permissive- /fp:except /Zc:preprocessor -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=0 /permissive-"
+ASAN	PM_CL="/MT /permissive- -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=0 /permissive- /analyze:only /analyze:autolog-"
+ASAN	PM_CL="/MT /permissive- /analyze:only /analyze:autolog- -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MT /D_ITERATOR_DEBUG_LEVEL=1 /permissive-"
+# No corresponding ASAN config, since the above differs from another config only in IDL
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=0 /permissive- /fp:strict"
+ASAN	PM_CL="/MTd /permissive- /fp:strict -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=1 /permissive-"
+ASAN	PM_CL="/MTd /permissive- -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=2 /permissive"
+ASAN	PM_CL="/MTd /permissive -fsanitize=address /Zi" PM_LINK="/debug"
 PM_CL="/MTd /D_ITERATOR_DEBUG_LEVEL=2 /permissive- /analyze:only /analyze:autolog-"
-# PM_CL="/permissive- /BE /c /MD"
-# PM_CL="/permissive- /BE /c /MTd"
+ASAN	PM_CL="/MTd /permissive- /analyze:only /analyze:autolog- -fsanitize=address /Zi" PM_LINK="/debug"
+# PM_CL="/permissive- /BE /c /MD" # TRANSITION, GH-395
+# PM_CL="/permissive- /BE /c /MTd" # TRANSITION, GH-395

```

Comment:
Ack, sorry - I left these transition comment cleanups in the concepts matrices. As there are very few such lines, I think another push would add more noise than the value of cleaning them up. Shout if you disagree.